SEED-platform/seed
# Fixes HTML error in sub-org submit button

#### What's this PR do?

Fixes the "Create sub-org" submit button. Lost the close tag on the line above while adding the translate tags in 1931c42f1.

#### How should this be manually tested?

Go to `/app/#/accounts/<org_id>/sub_org` and hit "Create a new sub-organization".

Expected: bootstrap primary button should now show correctly (won't without this patch).
